File: Python/2766.py
# Problema: 2766 - Entrada e Saída Lendo e Pulando Nomes | Resposta: Accepted
# Linguagem: Python 3.11 [+1s]                           | Tempo: 0.005s

linhas = [input() for _ in range(10)]
print(*[linhas[i] for i in [2,6,8]], sep='\n')

File: JavaScript/2766.js
// Problema: 2766 - Entrada e Saída Lendo e Pulando Nomes | Resposta: Accepted
// Linguagem: JavaScript 12.18 [+2s]                      | Tempo: 0.048s

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

console.log([lines[2], lines[6], lines[8]].join('\n'));